pkgs/development/python-modules/pythonocc-core/default.nix

```
@@ -12,11 +13,17 @@ stdenv.mkDerivation rec {
     sha256 = "1jk4y7f75z9lyawffpfkr50qw5452xzi1imcdlw9pdvf4i0y86k3";
   };
 
-  nativeBuildInputs = [ cmake swig ninja ];
+  postPatch = ''
```

should probably limit how many rebuilds this causes
```suggestion
  postPatch = stdenv.lib.optionalString stdenv.isDarwin ''
```

---

This only rebuilds this python package and one depended one which I am going to amend because it failed in nixpkgs-review.